src/hls/model/dfcir/lib/dfcir/DFCIROperations.cpp

```diff
@@ -0,0 +1,585 @@
+#include "dfcir/DFCIRDialect.h"
+#include "dfcir/DFCIROperations.h"
+
+::mlir::ParseResult
+mlir::dfcir::ScalarInputOp::parse(::mlir::OpAsmParser &parser,
+                                  ::mlir::OperationState &result) {
+  ::mlir::Type resRawTypes[1];
+  ::llvm::ArrayRef<::mlir::Type> resTypes(resRawTypes);
+  ::mlir::StringAttr nameAttr;
+  if (parser.parseLess())
+    return ::mlir::failure();
```

Comment:
Surround with "{ ... }"

Reply:
Ah, you said that you prefer to make one-line "if" statement in such case. Ok, so, do it!

Reply:
Fixed.